### ISSUE TITLE
`anchorpy client-gen` fails with error: "unknown variant `alias`, expected `struct` or `enum`"

### ISSUE THREAD

**Issue by totlsota** [kevinheavey/anchorpy#131]
using [raydium idl](https://solscan.io/account/CAMMCzo5YL8w4VFF8KVHrK22GGUsp5VTaW7grrKgrWqK#anchorProgramIDL)

`anchorpy client-gen` fails with error: "unknown variant `alias`, expected `struct` or `enum`"

when encountering this in the `types` section:
```json
{"name":"TickArryBitmap","type":{"kind":"alias","value":{"array":["u64",8]}}}
```

this error is raised:

```bash
/anchorpy/cli.py", line 109, in client_gen
    idl_obj = Idl.from_json(idl.read_text())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
solders.SerdeJSONError: unknown variant `alias`, expected `struct` or `enum` at line 1 column 20942
```

using latest version of anchorpy

user error is not out of the realm of possibility ;)

**Comment by totlsota** [comment]
[LOL :laughing: ](https://twitter.com/dj_d_sol/status/1740500808297287976)

**Comment by kevinheavey** [comment]
Fixed by #132
